src/Tizen.Wearable.CircularUI.Forms/ContentButton.cs

```diff
@@ -0,0 +1,175 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Tizen.Wearable.CircularUI.Forms
+{
+    /// <summary>
+    /// The ContentButton is a Button, which allows you to customize the View to be displayed.
+    /// </summary>
+    /// <since_tizen> 4 </since_tizen>
+    public class ContentButton : View, IButtonController
+    {
+        /// <summary>
+        /// BindableProperty. Identifies the View bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty ContentProperty = BindableProperty.Create(nameof(Content), typeof(View), typeof(ContentButton), default(View));
```

Comment:
Need to set parent when propertyChanged, 
Need to SetInheritedBindingContext , 
You can refer ContentView class